swagger-spec/licenses/list.yaml

```diff
@@ -1 +1,51 @@
 # /licenses/
+get:
+  summary: List all licenses
+  description: >-
+    A paginated list of licenses.
+    The returned preprints are sorted by their creation date, with the most recent
+    preprints appearing first.
+
+    #### Returns
+
+    A dictionary with a `data` and `links` property.
+
+
+    The `data` property contains an array of 10 licenses.
+    Each resource in the array is a separate license object.
```

Comment:
Can you make a note somewhere around here that the full representation of each license is returned (and therefore additional requests to license detail are not necessary)? Nodes has something [similar](https://github.com/caseyrollins/swagger-ui/blob/master/swagger-spec/nodes/list.yaml#L16).